services/secret-service/doc/openapi.yaml

```diff
@@ -49,19 +51,15 @@ paths:
           application/json:
             schema:
               type: "object"
-              properties:
-                data:
-                  $ref: "#/components/schemas/MutableAuthClient"
+              $ref: "#/components/schemas/MutableAuthClient"
```

Comment:
According to [API Guidelines](https://github.com/openintegrationhub/openintegrationhub/blob/master/Guidelines/ServiceAPIGuidelines.md) we return `data` object in the root of the response.

Reply:
It looks superfluous for a single object, as we haven't discussed any meta properties on a single object. But since we have agreed in the API guidelines, we will modify the Response objects.